nomad/structs/secure_variables.go

```diff
@@ -167,12 +167,12 @@ type SecureVariablesQuota struct {
 }
 
 type SecureVariablesUpsertRequest struct {
-	Data []*SecureVariable
+	Data []*SecureVariableDecrypted
 	WriteRequest
 }
 
-type SecureVariableUpsertRequest struct {
-	Data *SecureVariable
+type SecureVariablesEncryptedUpsertRequest struct {
+	Data []*SecureVariableEncrypted
 	WriteRequest
 }
```

Comment:
Another nice side-effect here is how it forces the split in behavior of these two requests to be more obvious.